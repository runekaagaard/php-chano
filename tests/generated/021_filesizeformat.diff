001- 1.0 KB
001+ 1.0 KiB