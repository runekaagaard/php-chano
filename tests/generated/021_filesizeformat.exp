1.0 KB