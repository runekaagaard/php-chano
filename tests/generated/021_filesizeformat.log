
---- EXPECTED OUTPUT
1.0 KB
---- ACTUAL OUTPUT
1.0 KiB
---- FAILED
