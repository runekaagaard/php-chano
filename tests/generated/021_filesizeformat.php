<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 1024)));
foreach ($items as $i) echo $i->input->filesizeformat();
