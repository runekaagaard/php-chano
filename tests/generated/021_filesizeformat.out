1.0 KiB